internal/blueprint/blueprint.go

```diff
@@ -103,6 +121,14 @@ func (b *Blueprint) GetPackages() []string {
 	for _, group := range b.Groups {
 		packages = append(packages, "@"+group.Name)
 	}
+
+	if kc := b.Customizations.GetKernel(); kc != nil && kc.Name != "" {
+		kpkg := Package{Name: b.Customizations.Kernel.Name}
+		packages = append(packages, kpkg.ToNameVersion())
+	} else if !b.hasKernelPackage() { // no Kernel specified; add default (unless already in packages)
```

Comment:
I think this might make it more fragile than we need it to be. We are not able to strictly verify the correct kernel names anyway, so I would say that the `hasKernelPackage()` logic can be skipped. The kernel should be configured in the blueprint, and if you do not, you get the same behavior as before this change.

Reply:
Figured it would be good for transition to be able to specify a kernel in the package list instead of the customization, but you're right that it's not particularly robust the way it detects it.

Reply:
Not specifying a kernel in the customization now adds the default "kernel" unconditionally.